### DISCUSSION TITLE
latest, snapshot or mqtt?

### DISCUSSION THREAD

**Discussion by brunopiras** [skrashevich/double-take#140] [Q&A] [answered]
Hello everyone,

I noticed that I get all three images, both latest and snapshot and mqtt, I would like to leave only mqtt, how can I do it from double-take?

Thanks

**Comment by brunopiras** [comment, marked as answer]
Auto answer my-self, set to '0' latest and snapshot.

**Reply by setterlee** [reply]
Thanks for sharing the solution
